my-app/src/components/HomesGuestLoves/Container.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { SwiperSlide } from 'swiper/react';
+import HomesGuestLovesHeader from './HomesGuestLovesHeader';
+import Carousel from './Carousel';
+import 'swiper/swiper-bundle.css';
+import '../../style.css';
+import Hotel from './Hotel';
+import data from '../../constants/content';
+
+const Container = (props) => {
+  return (
+    <div className={props.className}>
```

Comment:
you can take className variable from props like 
const Container = ({className}) =>{